CIScripts/Test/Tests/Integration/SNAT.Tests.ps1

```diff
@@ -0,0 +1,13 @@
+Describe "SNAT manual setup" {
```

Comment:
[opinion] We're not sure what this test does, but perhaps it should be moved to Functional?

Reply:
I removed Functional entirely for now